include/openssl/ssl.h

```diff
@@ -1628,7 +1629,7 @@ __owur STACK_OF(SSL_CIPHER) *SSL_get1_supported_ciphers(SSL *s);
 
 __owur int SSL_do_handshake(SSL *s);
 int SSL_renegotiate(SSL *s);
-__owur int SSL_renegotiate_abbreviated(SSL *s);
```

Comment:
I brought it into line with the preceding declaration of `SSL_renegotiate()`. At another point in this PR, I have this code:
````
    if (SSL_IS_DTLS(s))
        SSL_renegotiate(s);
    else
        SSL_renegotiate_abbreviated(s);
````
This was barfing with a compilation failure on the "else", but it was perfectly ok with the `SSL_rengotiate()` call. In truth, we really don't care what the return value is. If it fails we're not going to do anything different. I think that will be fairly common, so therefore having `__owur` doesn't seem appropriate.